imputation/ts_impute_test.go

```diff
@@ -22,11 +23,71 @@ func computeMedian(s TimeSeries) float64 {
 		values = append(values, v)
 	}
 	sort.Float64s(values)
-    if len(values) % 2 == 0{
-        return (values[len(values)/2] + values[len(values)/2 -1])/2
-    }else{
-	    return values[len(values)/2]
-    }
+	if len(values)%2 == 0 {
+		return (values[len(values)/2] + values[len(values)/2-1]) / 2
+	} else {
+		return values[len(values)/2]
+	}
+}
+
+// Values are produced using Python, Pandas linear interpolation method:
+func linearInterpolate(date string) TimeSeries {
+
+	// "length 3, years"
+	yseries := TimeSeries{
```

Comment:
We should test the "zero" and "one" cases, too.

Reply:
They are being tested, just that these cases don't need precomputed outputs.